config/config_prod.yaml

```diff
@@ -0,0 +1,63 @@
+log_directory: /report_service/logs/final
+#log_directory: /home/kaipak/datasets/report_service/logs/final
+profile_name: saml-pub
+s3_bucket: pds-web-analytics-cloud
+s3_logdir: logs
+#s3_logdir: logs_dev
```

Comment:
Ah this time the `s3_logdir` is `logs`, not `logs_dev` as in the above file. Let's double-check this.